Akash/html/assign 4/assign3.html

```diff
@@ -2,17 +2,11 @@
 <html>
     <head>
         <style>
-            table,
-            th,
-            td {
-                border: 2px solid grey;
-            }
-
-            th,
-            td {
-                text-align: center;
-                border: 2px solid black;
-            }
+            table, 
```

Comment:
this too also should be in a single line like table, th, td { watever style u want to write} in this way